### PR TITLE
Cursed #2 Yes.

### PR DESCRIPTION
**Pull request by alexyy802**

### CODE REVIEW

**Comment by alexyy802** [comment]
basically copied the example from the example folder to make the README.md more epik and also added da creators that are so pog omg

File: README.md [hunk 1]
```diff
@@ -51,3 +49,85 @@ To use our Voice Options Install
...
 ```py
 pip install -U pycord-utils[voice]
+
+```
+
+### Example
+```py
+import discord
+import pycord
+from pycord.ext import audio
```
Comment (RPSMain) [inline README.md:59]:
```suggestion
from pycord.ext import audio
```

File: README.md [hunk 2]
```diff
@@ -51,3 +49,85 @@ To use our Voice Options Install
...
+from discord.ext import commands
+
+
+class Bot(commands.Bot):
```
Comment (RPSMain) [inline README.md:63]: maybe remove the bot thing?

File: README.md [hunk 3]
```diff
@@ -51,3 +49,85 @@ To use our Voice Options Install
...
+    def __init__(self):
+        super(Bot, self).__init__(command_prefix=["audio ", "wave ", "aw "])
+
+        self.add_cog(Music(self))
+
+    async def on_ready(self):
+        print(f"Logged in as {self.user.name} | {self.user.id}")
+
+
+class Music(commands.Cog):
```
Comment (RPSMain) [inline README.md:73]: make this not a cog

**Review by RPSMain — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -51,3 +49,85 @@ To use our Voice Options Install
...
+        await player.play(tracks[0])
+
+
+bot = Bot()
+bot.run("TOKEN")
+```
+More examples can be found in the examples folder.
+
+
+### Creators
```
Comment (RPSMain) [inline README.md:131]: this list would become too long i don't think it should be here

**Review by RPSMain — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alexyy802) [reply README.md:59]: oki

**Review by RPSMain — APPROVED** [review]
might wanna add more

**Pull request merged**